.circleci/build-workbase-win/build-workbase-win.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+
+import os
+import subprocess
```

Comment:
we use `subprocess` alot, and the name is quite long. what if we do `import subprocess as sp` so the code can be more concise?

Reply:
@lolski fixed in https://github.com/graknlabs/workbase/pull/7/commits/84d585bce51fe8cf3eb26d777c4d9310c20e11fa